### DISCUSSION TITLE
How to upload a batch of files into a MerMEId instance

### DISCUSSION THREAD

**Discussion by peterstadler** [Edirom/MerMEId#99] [Q&A]
Let's say, we already have 3.000 MEI files that we'd want to display/edit with MerMEId …

**Comment by peterstadler** [comment]
@simar0at answered on Slack: 

> All data files are stored in a (the) exist-db running MerMEId. So all means of accessing and uploading files of exist are at your disposal. Including say WebDAV.

**Comment by peterstadler** [comment]
When running MerMEId via Docker, the eXistdb dashboard for all administrative tasks is available at `/apps/dashboard/`, e.g. `http://localhost:8080/apps/dashboard/`

**Reply by simar0at** [reply]
Please just keep in mind that unfortunately you can not login in dashboard using the login link in the top right corner. We had to move exist from /exist to / and that is no longer supported in dashboard.
You can log in in eXide or Monex. After that you can use `http://localhost:8080/apps/dashboard/admin` (using this link directly might display a log in form that works)

**Comment by simar0at** [comment]
To upload these files could be packaged as a XAR for the mermeid-data (pseudo) eXist application and installed this way.
It is also possible to upload data using WebDAV. On Windows I suggest [Cyberduck](https://cyberduck.io/), on Linux and MacOS the built in WebDAV connector should work (at least with the `edirom/mermeid:latest-java11-ShenGC`) variety).
![image](https://user-images.githubusercontent.com/641296/153908054-4288b368-a6af-4d93-8491-84cfd716ad69.png)
It is probably necessary to adjust the access rights for the files after upload.

**Comment by peterstadler** [comment]
I just added a How To [Uploading existing MEI files via eXide](https://github.com/Edirom/MerMEId/wiki/Uploading-existing-MEI-files-via-eXide) to our Wiki
